homeassistant/components/rpi_gpio/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Support for controlling GPIO pins of a Raspberry Pi."""
 import logging
 
+from RPi import GPIO  # pylint: disable=import-error
 from homeassistant.const import EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP
```

Comment:
Please add a blank line between 3rd party and homeassistant import groups. Preferably use isort to sort and group imports automatically.